cmdstanpy/stanfit.py

```diff
@@ -708,16 +723,23 @@ def draws_pd(
                 if not (param in self._column_names or param in pnames_base):
                     raise ValueError('unknown parameter: {}'.format(param))
         self._assemble_draws()
-        if self._draws_pd is None:
+
+        if inc_warmup and not self._save_warmup:
+            self._logger.warning(
+                'draws from warmup iterations not available,'
+                ' must run sampler with "save_warmup=True".'
+            )
+
+        num_draws = self._draws_sampling
+        if inc_warmup and self._save_warmup:
+            num_draws += self._draws_warmup
```

Comment:
Is `self._draws_warmup` always a number (I think it is, just making sure)

Reply:
yes, because at line 725 there's a call to `self._assemble_draws()`